kroxylicious-systemtests/src/test/java/io/kroxylicious/systemtests/AcceptanceST.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright Kroxylicious Authors.
+ *
+ * Licensed under the Apache Software License version 2.0, available at http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+package io.kroxylicious.systemtests;
+
+import java.io.IOException;
+import java.time.Duration;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestInfo;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import io.kroxylicious.systemtests.installation.kroxylicious.Kroxy;
+import io.kroxylicious.systemtests.steps.KafkaSteps;
+import io.kroxylicious.systemtests.steps.KroxySteps;
+import io.kroxylicious.systemtests.templates.strimzi.KafkaTemplates;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * The type Acceptance st.
+ */
+class AcceptanceST extends AbstractST {
+    private static final Logger LOGGER = LoggerFactory.getLogger(AcceptanceST.class);
+    private final String clusterName = "my-cluster";
+
+    /**
+     * Produce and consume message.
+     *
+     * @param testInfo the test info
+     * @throws IOException the io exception
+     */
+    @Test
+    void produceAndConsumeMessage(TestInfo testInfo) throws IOException {
```

Comment:
I'd like to see the test accept a Namespace parameter would be populated with an ephemeral namespace created by the test harness for the test.  That's the namespace where the test would create its resources.   After the test is finished, the test harness would delete the namespace.  All the resources would go automatically.  You'd use a Junit extension to do this (@ExtendWith etc).

Reply:
done